bigtable/nox.py

```diff
@@ -0,0 +1,85 @@
+# Copyright 2016 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+import nox
+
+
+@nox.session
+@nox.parametrize('python_version', ['2.7', '3.4', '3.5', '3.6'])
+def unit_tests(session, python_version):
+    """Run the unit test suite."""
+
+    # Run unit tests against all supported versions of Python.
+    session.interpreter = 'python%s' % python_version
+
+    # Install all test dependencies, then install this package in-place.
+    session.chdir(os.path.dirname(__file__))
+    session.install('mock', 'pytest', 'pytest-cov', '../core/')
+    session.install('-e', '.')
+
+    # Run py.test against the unit tests.
+    session.run('py.test', '--quiet',
+        '--cov=google.cloud.bigtable', '--cov=tests.unit', '--cov-append',
+        '--cov-config=.coveragerc', '--cov-report=', '--cov-fail-under=97',
```

Comment:
Agreed in principle, but it defaults to 100, so I needed _something_. I suppose I could make it zero. On the other hand, if excessively little is being covered in any given version, that is probably an indicator of a problem, too.